src/tests/features/rpfilter.at

```diff
@@ -2,21 +2,23 @@ FWD_START_TEST([rpfilter - strict])
 AT_KEYWORDS(rpfilter)
 CHECK_NFTABLES_FIB()
 
+AT_CHECK([sed -i 's/^LogDenied.*/LogDenied=multicast/' ./firewalld.conf])
 AT_CHECK([sed -i 's/^IPv6_rpfilter.*/IPv6_rpfilter=strict/' ./firewalld.conf])
 FWD_RELOAD()
 
 NFT_LIST_RULES([inet], [filter_PREROUTING], 0, [dnl
     table inet firewalld {
         chain filter_PREROUTING {
             icmpv6 type { nd-router-advert, nd-neighbor-solicit } accept
-            meta nfproto ipv6 fib saddr . mark . iif oif missing drop
+            meta nfproto ipv6 fib saddr . mark . iif oif missing meta pkttype multicast log prefix "rpfilter_DROP: " drop
```

Comment:
Won't this cause it to drop only for multicast packets? Where as `LogDenied` is only about logging.

To support this, I think it would have to be two separate rules.

Reply:
Correct, I apologize! I’ve now separated the rule configurations for cases where log_denied is not equal to 'all', and created distinct test cases to specifically validate each scenario.